libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts

```diff
@@ -223,12 +223,16 @@ export class TigerExecutionResult implements IExecutionResult {
                 : {}),
         };
 
+        const useExecutionResult = !options.visualizationObjectId ? this.resultId : undefined;
```

Comment:
@dremileta I find this a bit confusing, based on the variable name, I'd expect it is a boolean, but this is either the resultId or undefined.

Maybe we don't need it actually looking at it now, and we can use the vis obj id directly  with a comment

```ts
executionResult: !!options.visualizationObjectId ? undefined : this.resultId, // use the visualizationObject for the export instead of the execution when provided
visualizationObject: options.visualizationObjectId 
```

Reply:
Yeah, I was trying to think what you meant by having variable there, and this what I came up with. But the naming is indeed weird

Reply:
yep, sorry for the confusion, now it's ok.